includes/wf_crm_webform_base.inc

```diff
@@ -85,6 +85,10 @@ abstract class wf_crm_webform_base {
           $params['options']['sort'] = 'is_primary DESC';
         }
         $result = wf_civicrm_api($ent, 'get', $params);
+        // Handle location field sorting
+        if(in_array($ent, wf_crm_location_fields())){
+          $result['values'] = $this->reorderByLocationType($ent, $result['values']);
```

Comment:
This looks like the right change here. However (see comments below) I think the function isn't yet doing what we want it to do.